src/movies.js

```diff
@@ -4,10 +4,56 @@
 
 // Get the average of all rates with 2 decimals 
 
+ratesAverage = function(arr){
+  var ratesArr = arr.map((arr) => {return arr.rate});
+
+  var ratesSum = ratesArr.reduce(function(accumulator, currentNum){
+    return accumulator + currentNum;
+  });
+
+  var average = ratesSum / ratesArr.length;
```

Comment:
Really good code here Chloé :) great separation of concerns and really cleanly set out :) Great job